packages/bot-skeleton/src/services/tradeEngine/trade/Balance.js

```diff
@@ -22,17 +22,10 @@ export default Engine =>
 
         // eslint-disable-next-line class-methods-use-this
         getBalance(type) {
-            const { scope } = this.store.getState();
             const { client } = DBotStore.instance;
             const balance = (client && client.balance) || 0;
-            let value = balance;
 
-            if (scope === 'BEFORE_PURCHASE') {
```

Comment:
So don't we need anymore to know if it is before purchase or not ?

Reply:
@sara-fs @sandeep-deriv According to the commentary described for what it was done, you can understand that if we have 'BEFORE_PURCHASE', then we have adjusted the correct balance figure that we display in the Journal (notifications) panel to the one that we see in the upper right corner of the application. When commenting on this piece of code, the bug was removed, and the message with the balance before the start of the contract ('BEFORE_PURCHASE') in the Journal (notifications) corresponds to the balance in the upper right corner. At least when testing, I did not find a case where there could be an error.